wallet/src/actors/app/handlers/create_vtt.rs

```diff
@@ -128,6 +145,255 @@ pub fn validate_output_addresses(
     })
 }
 
+// Serialization helper
+
+/// Value transfer output transaction data structure
+#[derive(Debug, Eq, PartialEq, Clone, Serialize, Deserialize, Hash, Default)]
+pub struct VttOutputParamsHelper {
+    pub address: String,
+    #[serde(
+        serialize_with = "u64_to_string",
+        deserialize_with = "number_from_string"
+    )]
+    pub amount: u64,
+    pub time_lock: Option<u64>,
+}
+
+impl From<VttOutputParams> for VttOutputParamsHelper {
+    fn from(x: VttOutputParams) -> Self {
+        VttOutputParamsHelper {
+            address: x.address,
+            amount: x.amount,
+            time_lock: x.time_lock,
+        }
+    }
+}
+
+impl From<VttOutputParamsHelper> for VttOutputParams {
+    fn from(x: VttOutputParamsHelper) -> Self {
+        VttOutputParams {
+            address: x.address,
+            amount: x.amount,
+            time_lock: x.time_lock,
+        }
+    }
+}
+
+/// Transaction data structure
+#[derive(Debug, Eq, PartialEq, Clone, Serialize, Deserialize)]
+// FIXME(#649): Remove clippy skip error
+#[allow(clippy::large_enum_variant)]
+pub enum TransactionHelper {
+    #[serde(
+        serialize_with = "into_generic_type::<_, VTTransactionHelper, _>",
+        deserialize_with = "from_generic_type::<_, VTTransactionHelper, _>"
+    )]
+    ValueTransfer(VTTransaction),
+    DataRequest(DRTransaction),
+    Commit(CommitTransaction),
+    Reveal(RevealTransaction),
+    Tally(TallyTransaction),
+    Mint(MintTransaction),
+}
+
+impl From<Transaction> for TransactionHelper {
+    fn from(x: Transaction) -> Self {
+        match x {
+            Transaction::ValueTransfer(vttransaction) => {
```

Comment:
Isnt this repeated code? If so, can we just move it to a single file?

Reply:
Something like `wallet/src/serialization.rs`